### PR TITLE
DOCS-12051-Renew certificates for servers in a cluster or group

### PR DESCRIPTION
**Pull request by luanamulesoft**

### CODE REVIEW

**Comment by rodrigo-neira** [comment]
I have confirmed with the Engineering Team, that renewing certificates does not trigger a runtime restart. Please clarify this in the docs. Thanks!

File: modules/ROOT/pages/servers-cert-renewal.adoc [hunk 1]
```diff
@@ -47,17 +47,46 @@ You can renew a certificate for a specific server or multiple servers within a c
...
+. Check the radio button to select the server or servers you want to update.
+. From the *Actions* menu, select *Renew Certificate*.
++
+image::rtm-server-certificate-group-select-server.png[Select Server]
+. Select the checkbox to confirm your choice, and then click *Renew*.
++
+image::rtm-server-certificate-group-renew.png[Confirm Renewal]
+. Check *Certificate expiration date* to verify that the certificate renewed successfully.
+
+If, after following the steps previously described, the *Renew Certificate* option in the *Actions* pull-don menu is not available, you might need to xref:installing-and-configuring-runtime-manager-agent#update-the-agent.adoc[update your *Agent* version].
```
Comment (fpiacentini08) [inline modules/ROOT/pages/servers-cert-renewal.adoc]: typo pull-down

Comment (luanamulesoft) [reply modules/ROOT/pages/servers-cert-renewal.adoc]: Typo fixed! Thank you!! 😊

**Review by rodrigo-neira — DISMISSED** [review]

File: modules/ROOT/pages/servers-cert-renewal.adoc [hunk 2]
```diff
@@ -47,17 +47,46 @@ You can renew a certificate for a specific server or multiple servers within a c
 
 The status for the server must be `Running` to update its certificate.
 
-To renew a certificate for a server:
+To renew a certificate for a standalone server:
```
Comment (kevintroller) [inline modules/ROOT/pages/servers-cert-renewal.adoc:50]:
```suggestion
To renew a certificate for a standalone server, follow these steps:
```

File: modules/ROOT/pages/servers-cert-renewal.adoc [hunk 3]
```diff
@@ -47,17 +47,46 @@ You can renew a certificate for a specific server or multiple servers within a c
...
 . Click *Settings*.
 . From the *Actions* menu, select *Renew Certificate*.
++
+image::rtm-server-standalone-settings.png[Settings Tab]
 . Select the checkbox to confirm your choice, and then click *Renew*.
++
+image::rtm-server-standalone-renew.png[Confirm Renewal]
 . Check *Certificate expiration date* to verify that the certificate renewed successfully.
 
+To renew a certificate for servers in a cluster or group:
```
Comment (kevintroller) [inline modules/ROOT/pages/servers-cert-renewal.adoc:67]:
```suggestion
To renew a certificate for servers in a cluster or group, follow these steps:
```

File: modules/ROOT/pages/servers-cert-renewal.adoc [hunk 4]
```diff
@@ -47,17 +47,46 @@ You can renew a certificate for a specific server or multiple servers within a c
...
+. Check the radio button to select the server or servers you want to update.
+. From the *Actions* menu, select *Renew Certificate*.
++
+image::rtm-server-certificate-group-select-server.png[Select Server]
+. Select the checkbox to confirm your choice, and then click *Renew*.
++
+image::rtm-server-certificate-group-renew.png[Confirm Renewal]
+. Check *Certificate expiration date* to verify that the certificate renewed successfully.
+
+If, after following the steps previously described, the *Renew Certificate* option in the *Actions* pull-down menu is not available, you might need to xref:installing-and-configuring-runtime-manager-agent#update-the-agent.adoc[update your *Agent* version].
```
Comment (kevintroller) [inline modules/ROOT/pages/servers-cert-renewal.adoc]:
```suggestion
If the *Renew Certificate* option in the *Actions* pull-down menu is not available after performing the previous steps, you may need to xref:installing-and-configuring-runtime-manager-agent#update-the-agent.adoc[update your *Agent* version].
```

File: modules/ROOT/pages/servers-cert-renewal.adoc [hunk 5]
```diff
@@ -47,17 +47,46 @@ You can renew a certificate for a specific server or multiple servers within a c
...
+The *Server certificate renewal* feature, which enables you to renew server certificates directly from Runtime Manager, was introduced by Runtime Manager 2.8.4. This feature requires Runtime Manager Agent 1.10.0 or later.
+You can hover over the *Renew Certificate* option in the *Actions* pull-don menu, and a tooltip in the UI will tell you the minimum required Runtime Agent version.
```
Comment (kevintroller) [inline modules/ROOT/pages/servers-cert-renewal.adoc]:
```suggestion
You can hover over the *Renew Certificate* option in the *Actions* pull-down menu, and a tooltip in the UI will indicate you the minimum required Runtime Agent version.
```

**Review by kevintroller — DISMISSED** [review]
Added minor comments

**Review by kevintroller — APPROVED** [review]

**Pull request merged**